colorcalculator.php

```diff
@@ -0,0 +1,74 @@
+<html>
+	<body>
+		<h1> Color Calculator </h1> <br> <br>
+			<form action=""
+
+				<p>First Color
+						<select name="color1" class="color1">
+							<option value='0'>--Select function--</option>
```

Comment:
This is fine, but you don't really need to give it a value of `0`. you can just say `value=''`